extensions/wikia/AchievementsII/Ach_setup.php

```diff
@@ -1,199 +1,199 @@
 
 //dependencies
 $wgAutoloadClasses[ 'UploadAchievementsFromFile' ] = "{$dir}UploadAchievementsFromFile.class.php";
 $wgAutoloadClasses[ 'WikiaPhotoGalleryUpload' ] = "{$dir}../WikiaPhotoGallery/WikiaPhotoGalleryUpload.class.php";
 
 // I18N
-$wgExtensionMessagesFiles['AchievementsII'] = $dir.'AchievementsII.i18n.php';
-$wgExtensionMessagesFiles['AchievementsIIAliases'] = $dir.'AchievementsII.alias.php' ;
+$wgExtensionMessagesFiles[ 'AchievementsII' ] = $dir . 'AchievementsII.i18n.php';
+$wgExtensionMessagesFiles[ 'AchievementsIIAliases' ] = $dir . 'AchievementsII.alias.php';
 
 // Michał Roszka (Mix) <michal@wikia-inc.com>
 // BugId:10474
 // I want the code of the extension to be included regardless of the $wgEnableAchievementsExt.
 // The effective execution still depends on $wgEnableAchievementsExt's value.
 // Also: grep CommonExtensions.php for BugId:10474
 if( !empty( $wgEnableAchievementsExt ) ) {
-    // SETUP
-    $wgExtensionFunctions[] = 'Ach_Setup';
+	// SETUP
+	$wgExtensionFunctions[] = 'Ach_Setup';
 }
 // AJAX
 $wgAjaxExportList[] = 'AchAjax';
 
 function Ach_Setup() {
-	wfProfileIn(__METHOD__);
+	wfProfileIn( __METHOD__ );
 
 	global $wgHooks;
-	$wgHooks['ArticleSaveComplete'][] = 'Ach_ArticleSaveComplete';
-	$wgHooks['GetHTMLAfterBody'][] = 'Ach_GetHTMLAfterBody';
-	$wgHooks['UploadVerification'][] = 'Ach_UploadVerification';
+	$wgHooks[ 'ArticleSaveComplete' ][] = 'Ach_ArticleSaveComplete';
+	$wgHooks[ 'GetHTMLAfterBody' ][] = 'Ach_GetHTMLAfterBody';
+	$wgHooks[ 'UploadVerification' ][] = 'Ach_UploadVerification';
 
 	//hooks for user preferences
-	$wgHooks['GetPreferences'][] = 'Ach_UserPreferences';
-	$wgHooks['MonacoSidebarGetMenu'][] = 'Ach_GetMenu';
+	$wgHooks[ 'GetPreferences' ][] = 'Ach_UserPreferences';
+	$wgHooks[ 'MonacoSidebarGetMenu' ][] = 'Ach_GetMenu';
 
 	//hook for purging Achievemets-related cache
-	$wgHooks['AchievementsInvalidateCache'][] = 'Ach_InvalidateCache';
+	$wgHooks[ 'AchievementsInvalidateCache' ][] = 'Ach_InvalidateCache';
 
-	wfProfileOut(__METHOD__);
+	wfProfileOut( __METHOD__ );
 }
 
-function Ach_GetMenu(&$nodes) {
-	$nodes[0]['children'][] = count($nodes);
+function Ach_GetMenu( &$nodes ) {
+	$nodes[ 0 ][ 'children' ][] = count( $nodes );
 	$nodes[] = array(
 		//'original' => 'achievementsleaderboard',
 		//the message is stored in /languages/messages/wikia/MessagesEn.php to avoid loading the i18n for the extension
-		'text' => wfMsg('achievements-leaderboard-navigation-item'),
-		'href' => Skin::makeSpecialUrl("Leaderboard"),
-  		//'depth' => 1,
+		'text' => wfMsg( 'achievements-leaderboard-navigation-item' ),
+		'href' => Skin::makeSpecialUrl( "Leaderboard" ),
+		//'depth' => 1,
 		//'parentIndex' => 0
 	);
 
 	return true;
 }
 
-function Ach_UploadVerification($destName, $tempPath, &$error) {
-	if (Ach_isBadgeImage($destName, true /* check user right to upload sponsored badge */ )) {
+function Ach_UploadVerification( $destName, $tempPath, &$error ) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove the unused function parameter "$tempPath". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1172)